clone-replace.php

```diff
@@ -143,4 +145,62 @@ function cr_the_status( $post ) {
 		}
 	}
 
+	function cr_maybe_enqueue_gutenberg_script() {
+		wp_enqueue_script(
+			'clone-replace',
+			plugins_url( '/clone-replace/build/cloneReplace.js', __DIR__ ),
+			[ 'wp-blocks', 'wp-i18n' ],
+			'1.0.0',
```

Comment:
We are using the `wpAssets.js` file to make an asset manifest that includes hash versions for scripts in order to automatically bust cache. This function should leverage that file in order to load the appropriate hash in the version parameter here. Reference `wp-starter-plugin` for implementation details.